packages/components/releaseNotes/components.md

```diff
@@ -5,6 +5,11 @@ Components, models, actions, and utility functions for LabKey applications and p
 *Released*: 12 July 2022
 Added SearchScope enum and ContainerFilter mapping utility method
 
+### version TBD
+*Released*: TBD
+* Issue 45815: Show view name in tabbed export modal
+* Issue 45686: Change "Create Samples" to "Add Samples" on dashboard button
```

Comment:
Should SamplesAddButton's default text be updated to "Add Samples"?

Reply:
Good question. Currently we have one usage per app that wants "Create Samples" and one that wants "Add", so it's kind of a toss-up.